---
layout: page
permalink: /teaching/
title: Resources
description:
nav: true
nav_order: 4
---

## **Bioinformatics Training Resources**

<hr style="border: 3px solid #00008B;">


### **1.1 Taylor Paisie's VEME 2024 NGS De Novo Assembly Tutorial**  

This course focuses on <em>de novo</em> genome assembly using next-generation sequencing (NGS) data, providing a comprehensive overview of workflows and practical exercises tailored for the VEME 2024 workshop. It includes step-by-step guidance on assembling viral genomes, with specific tools and methods to handle Illumina sequencing data efficiently.


**<span style="color: #00A3E0; font-weight: bold;">Course Link:</span>** [Taylor Paisie's VEME 2024 NGS De Novo Assembly](https://github.com/taylorpaisie/VEME_2024_NGS_Denovo_Assembly)

<hr style="border: 1px dotted rgba(0, 0, 139, 0.2);">

### **1.2 Taylor Paisie's VEME 2024 NGS Variant Calling**  

This course provides comprehensive guidance on variant calling workflows using next-generation sequencing (NGS) data. It is tailored for the VEME 2024 workshop and includes step-by-step instructions and examples.

**<span style="color: #00A3E0; font-weight: bold;">Course Link:</span>** [Taylor Paisie's VEME 2024 NGS Variant Calling](https://github.com/taylorpaisie/VEME_2024_NGS_Variant_Calling)

<hr style="border: 3px solid #00008B;">

## **2. Broad Institute’s Training Resources**

### **2.1 Viral Genomics Workshops**  

These workshops provide hands-on training in genomic analysis workflows, focusing on the study of viral genomes using next-generation sequencing (NGS) data. Hosted by the Broad Institute, they offer practical insights into cutting-edge tools and techniques for viral surveillance, assembly, and data interpretation.

**<span style="color: #00A3E0; font-weight: bold;">Course Link:</span>** [Viral Genomics Workshops](https://broadinstitute.github.io/viral-workshops/)

<hr style="border: 1px dotted rgba(0, 0, 139, 0.2);">

### **2.2 Viral De Novo Assembly Workflows on the Terra Cloud Platform**  

This course focuses on viral <em>de novo<em> assembly workflows using Illumina sequencing data. It specifically uses Lassa virus data as a case study and leverages the Terra cloud platform to demonstrate the assembly process.

**<span style="color: #00A3E0; font-weight: bold;">Course Link:</span>** [Viral De Novo Assembly Workflows on Terra Cloud Platform](https://broadinstitute.github.io/viral-workshops/veme-ngs/denovo.html#viral-de-novo-assembly)

<hr style="border: 3px solid #00008B;">

## **3. A Beginner’s Guide to Genomic Data Analysis: Variant Calling**  

The link provides a beginner-friendly introduction to genomic data analysis, specifically focusing on variant calling, which involves identifying genetic variations in sequencing data. It outlines the key steps in the process, including quality control, alignment of reads to a reference genome, and the use of tools like GATK for variant identification and filtering. The guide emphasizes the importance of understanding the pipeline and highlights resources and tools to help newcomers get started with genomic data analysis.

**<span style="color: #00A3E0; font-weight: bold;">Course Link:</span>** [A Beginner’s Guide to Genomic Data Analysis: Variant Calling](https://medium.com/@manabeel.vet/a-beginners-guide-to-genomic-data-analysis-variant-calling-ad8515eebddf)

<hr style="border: 3px solid #00008B;">
