<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    
    <!-- Website verification -->
    <meta name="google-site-verification" content="">
<!-- Avoid warning on Google Chrome
        Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'interest-cohort'.
        see https://stackoverflow.com/a/75119417
    -->
    <meta http-equiv="Permissions-Policy" content="interest-cohort=()">

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Gerald  Mboowa</title>
    <meta name="author" content="Gerald  Mboowa">
    <meta name="description" content="Gerald Mboowa integrates data science, genomics, and bioinformatics to tackle critical global health challenges, driving innovations in antimicrobial resistance surveillance, outbreak investigation, and pathogen characterization.
">
    <meta name="keywords" content="academic-website, gerald-mboowa, mboowa">


    <!-- Bootstrap & MDB -->
    <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
    <!-- <link rel="stylesheet" href="/assets/css/mdb.min.css"> -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Bootstrap Table -->
    <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="/assets/css/academicons.min.css">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css" media="" id="highlight_theme_light">

    

    <!-- Styles -->
    
    <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="http://localhost:4000/">

    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css" media="none" id="highlight_theme_dark">
    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    


    <!-- Inline CSS for Light/Dark Mode -->
    <style>
      :root {
          --background-color: #FFFFFF;
          --text-color: #1A1A1A;
          --link-color: #8B008B;
      }

      [data-theme="dark"] {
          --background-color: #121212;
          --text-color: #ffffff;
          --link-color: #4da8da;
      }

      body {
          background-color: var(--background-color);
          color: var(--text-color);
      }

      a {
          color: var(--link-color);
      }

      #theme-toggle {
          position: fixed;
          top: 10px;
          right: 10px;
          z-index: 1000;
          background-color: var(--background-color);
          color: var(--text-color);
          border: 1px solid var(--text-color);
          padding: 8px 12px;
          font-size: 14px;
          border-radius: 5px;
          cursor: pointer;
      }

      /* Font scaling for media & teaching pages */
      .media-page h1,
      .teaching-page h1 {
          font-size: 2rem;
          margin-top: 1.2rem;
          margin-bottom: 0.6rem;
      }

      .media-page h2,
      .teaching-page h2 {
          font-size: 1.5rem;
          margin-top: 1rem;
          margin-bottom: 0.5rem;
      }

      .media-page h3,
      .teaching-page h3 {
          font-size: 1.2rem;
          margin-top: 0.8rem;
          margin-bottom: 0.4rem;
      }

      @media screen and (max-width: 768px) {
          .media-page h1,
          .teaching-page h1 {
              font-size: 1.6rem;
          }

          .media-page h2,
          .teaching-page h2 {
              font-size: 1.2rem;
          }

          .media-page h3,
          .teaching-page h3 {
              font-size: 1.05rem;
          }
      }

      @media screen and (max-width: 480px) {
          .media-page h1,
          .teaching-page h1 {
              font-size: 1.4rem;
          }

          .media-page h2,
          .teaching-page h2 {
              font-size: 1.1rem;
          }

          .media-page h3,
          .teaching-page h3 {
              font-size: 1rem;
          }
      }
    </style>
  </head>

  <!-- Body -->
  
  
  
  <body class="-page fixed-top-nav sticky-bottom-footer">

    <!-- Theme Toggle Button -->
    <button id="theme-toggle">ðŸŒ™ Toggle Theme</button>

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <!-- Social Icons -->
          <div class="navbar-brand social">
            <a href="mailto:%67%6D%62%6F%6F%77%61@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fas fa-envelope"></i></a>
            <a href="https://orcid.org/0000-0001-8445-9414" title="ORCID" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="ai ai-orcid"></i></a>
            <a href="https://scholar.google.com/citations?user=P9dSDhoAAAAJ" title="Google Scholar" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="ai ai-google-scholar"></i></a>
            <a href="https://www.researchgate.net/profile/Gerald-Mboowa/" title="ResearchGate" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="ai ai-researchgate"></i></a>
            <a href="https://github.com/gmboowa" title="GitHub" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="fab fa-github"></i></a>
            <a href="https://www.linkedin.com/in/gerald-mboowa-99020537" title="LinkedIn" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="fab fa-linkedin"></i></a>
            <a href="https://twitter.com/GeraldGmboowa" title="Twitter" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="fab fa-twitter"></i></a>
            

          </div>
          
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item active">
                <a class="nav-link" href="/">About<span class="sr-only">(current)</span></a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/publications/">Publications</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/repositories/">Repositories</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/projects/">Projects</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/teaching/">Resources</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/media/">Media</a>
              </li>

              <!-- Toogle theme mode -->
              <li class="toggle-container">
                <button id="light-toggle" title="Change theme">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </nav>

      <!-- Scrolling Progress Bar -->
      <progress id="progress" value="0">
        <div class="progress-container">
          <span class="progress-bar"></span>
        </div>
      </progress>
    </header>


    <!-- Content -->
    <div class="container mt-5">
      
        <!-- about.html -->
      <div class="post">
        <header class="post-header">
          <h1 class="post-title">
            <span class="font-weight-bold">Gerald</span>  Mboowa
          </h1>
          <p class="desc"></p>
        </header>

        <article>
          <div class="profile float-right">

              <figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/gerald-mboowa-1-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/gerald-mboowa-1-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/gerald-mboowa-1-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/gerald-mboowa-1.png" class="img-fluid z-depth-1 rounded" width="auto" height="auto" alt="gerald-mboowa-1.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">
  </picture>

</figure>

            <div class="more-info">
              <p></p>
<br> <p></p>

            </div>
          </div>

          <div class="clearfix">
            <p>Hi, Iâ€™m Gerald Mboowa, a scientist dedicated to advancing global health through innovative research and data-driven solutions, particularly in low- and middle-income countries (LMICs). My work integrates genome sequencing technologies and bioinformatics into public health strategies to address critical health challenges. As an implementation scientist, we have developed impactful bioinformatics pipelines, including <strong><a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000583" rel="external nofollow noopener noopener noreferrer" target="_blank">rMAP</a></strong> (Rapid Microbial Analysis Pipeline) for native Linux command-line environments (also supported on macOS) for rapid, end-to-end analysis of <strong>ESKAPE</strong> bacterial whole-genome sequencing data (from raw reads through core outputs such as genome assembly/annotation, AMR profiling &amp; strain/lineage characterization) and <strong><a href="https://academic.oup.com/bioinformaticsadvances/advance-article/doi/10.1093/bioadv/vbag046/8483022" rel="external nofollow noopener noopener noreferrer" target="_blank">rMAP 2.0</a></strong>, a modular &amp; reproducible WDLâ€“Cromwellâ€“Docker workflow that standardizes microbial whole-genome analysis of <strong>ESKAPEE</strong> pathogens by integrating read QC/trimming, assembly &amp; annotation, resistance/virulence/mobile-element profiling, sequence inference and phylogenetic reconstruction, and automated HTML reporting across laptops, institutional servers, and cloud environments.</p>

<p>We also developed <strong><a href="#">HIV-DRIVES</a></strong> (HIV Drug Resistance Identification, Variant Evaluation, and Surveillance Pipeline). These tools significantly enhance antimicrobial resistance (AMR) surveillance and pathogen characterization, supporting outbreak investigations and evidence-based public health decision-making.</p>

<p>The rMAP pipeline facilitates the rapid identification of clinically important bacterial pathogens and genomic determinants of resistance and virulence, offering actionable insights to develop targeted interventions and inform AMR policies. Meanwhile, HIV-DRIVES focuses on identifying HIV drug resistance, evaluating variants, and enabling robust surveillance. Together, these pipelines empower health systemsâ€”especially in LMICsâ€”with the capacity to utilize genomic data, transforming how public health challenges are addressed.</p>

<p>With expertise spanning next-generation sequencing (NGS) and genomic data analysis, I focus on bridging data science, bioinformatics, and genomics to develop innovative solutions for improving diagnostics, treatment strategies, and public health outcomes. My contributions underscore the transformative potential of bioinformatics in addressing complex biological systems and translating genomic insights into strategies that enhance global health.</p>

<p>Currently, I contribute to global health initiatives at the
<a href="https://www.broadinstitute.org/" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer" target="_blank">
Broad Institute of MIT and Harvard
</a>,
building on my extensive experience in developing bioinformatics frameworks for infectious disease surveillance. My work encompasses AMR, genomics, and infectious disease surveillance, with a strong focus on fostering equitable solutions for LMICs. Previously, I worked with public health institutions such as the
<a href="https://idi.mak.ac.ug/" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer" target="_blank">
Infectious Diseases Institute at Makerere University
</a>
and the
<a href="https://africacdc.org/" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer" target="_blank">
Africa Centres for Disease Control and Prevention (Africa CDC)
</a>
to implement genomic surveillance programs aimed at enhancing disease detection and response systems across Africa. I contributed to these efforts as a
<a href="https://africacdc.org/people/gerald-mboowa/" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer" target="_blank">
bioinformatics implementation expert
</a>.</p>

<p>Passionate about translating research into actionable public health outcomes, I actively support Genomics, Bioinformatics, and Data Science training programs at the
<a href="https://ace.ac.ug/" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer" target="_blank">
African Center of Excellence in Bioinformatics &amp; Data-intensive Sciences (ACE)
</a>
and
<a href="https://chs.mak.ac.ug/team/gerald-mboowa" target="_blank" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer">
Makerere University
</a>.</p>

<p>For more details about my research contributions, visit my
<a href="https://orcid.org/0000-0001-8445-9414" target="_blank" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer">
ORCID
</a>,
<a href="https://scholar.google.com/citations?user=P9dSDhoAAAAJ" target="_blank" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer">
Google Scholar
</a>,
and
<a href="https://www.researchgate.net/profile/Gerald-Mboowa" target="_blank" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer">
ResearchGate
</a>
pages. You can also connect with me via
<a href="https://twitter.com/GeraldGMboowa" target="_blank" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer">
Twitter
</a>
and
<a href="https://www.linkedin.com/in/gerald-mboowa-99020537/" target="_blank" style="font-weight:900;" rel="external nofollow noopener noopener noreferrer">
LinkedIn
</a>.</p>

<p>Letâ€™s connect and explore collaborative opportunities to advance global health, genomics, bioinformatics, data science, and public health innovation!</p>

          </div>

          <!-- News -->
          

          <!-- Latest posts -->
          

          <!-- Selected papers -->
          <h2>
          <a href="/publications/" style="color: inherit;">
            <strong>Selected Publications</strong> <!-- Bold text -->
          </a>
        </h2>
          <div class="publications">
            <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Science</abbr></div>
<!-- Entry bib key -->
        <div id="africacdc_2022" class="col-sm-8">
        <!-- Title -->
        <div class="title">The evolving SARS-CoV-2 epidemic in Africa: Insights from rapidly expanding genomic surveillance</div>
        <!-- Author -->
        <div class="author">
        

        Houriiyah Tegally,Â James E San,Â Matthew Cotton, and
          <span class="more-authors" title="click to view 399 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '399 more authors' ? 'Monika Moir, Bryan Tegomoh, Gerald Mboowa, Darren P Martin, Cheryl Baxter, Arnold W Lambisia, Amadou Diallo, Daniel G Amoako, Moussa M Diagne, Abay Sisay, Abdel-Rahman N Zekri, Abdou Salam Gueye, Abdoul K Sangare, Abdoul-Salam Ouedraogo, Abdourahmane Sow, Abdualmoniem O Musa, Abdul K Sesay, Abe G Abias, Adam I Elzagheid, Adamou Lagare, Adedotun-Sulaiman Kemi, Aden Elmi Abar, Adeniji A Johnson, Adeola Fowotade, Adeyemi O Oluwapelumi, Adrienne A Amuri, Agnes Juru, Ahmed Kandeil, Ahmed Mostafa, Ahmed Rebai, Ahmed Sayed, Akano Kazeem, Aladje Balde, Alan Christoffels, Alexander J Trotter, Allan Campbell, Alpha K Keita, Amadou Kone, Amal Bouzid, Amal Souissi, Ambrose Agweyu, Amel Naguib, Ana V Gutierrez, Anatole Nkeshimana, Andrew J Page, Anges Yadouleton, Anika Vinze, Anise N Happi, Anissa Chouikha, Arash Iranzadeh, Arisha Maharaj, Armel L Batchi-Bouyou, Arshad Ismail, Augustina A Sylverken, Augustine Goba, Ayoade Femi, Ayotunde E Sijuwola, Baba Marycelin, Babatunde L Salako, Bamidele S Oderinde, Bankole Bolajoko, Bassirou Diarra, Belinda L Herring, Benjamin Tsofa, Bernard Lekana-Douki, Bernard Mvula, Berthe-Marie Njanpop-Lafourcade, Blessing T Marondera, Bouh Abdi Khaireh, Bourema Kouriba, Bright Adu, Brigitte Pool, Bronwyn McInnis, Cara Brook, Carolyn Williamson, Cassien Nduwimana, Catherine Anscombe, Catherine B Pratt, Cathrine Scheepers, Chantal G Akoua-Koffi, Charles N Agoti, Chastel M Mapanguy, Cheikh Loucoubar, Chika K Onwuamah, Chikwe Ihekweazu, Christian N Malaka, Christophe Peyrefitte, Chukwa Grace, Chukwuma E Omoruyi, Clotaire D RafaÃ¯, Collins M Morangâ€™a, Cyril Erameh, Daniel B Lule, Daniel J Bridges, Daniel Mukadi-Bamuleka, Danny Park, David A Rasmussen, David Baker, David J Nokes, Deogratius Ssemwanga, Derek Tshiabuila, Dominic S Y Amuzu, Dominique Goedhals, Donald S Grant, Donwilliams O Omuoyo, Dorcas Maruapula, Dorcas W Wanjohi, Ebenezer Foster-Nyarko, Eddy K Lusamaki, Edgar Simulundu, Edidah M Ongâ€™era, Edith N Ngabana, Edward O Abworo, Edward Otieno, Edwin Shumba, Edwine Barasa, Bara El Ahmed, Elhadi A Ahmed, Emmanuel Lokilo, Enatha Mukantwari, Eromon Philomena, Essia Belarbi, Etienne Simon-Loriere, EtilÃ© A Anoh, Eusebio Manuel, Fabian Leendertz, Fahn M Taweh, Fares Wasfi, Fatma Abdelmoula, Faustinos T Takawira, Fawzi Derrar, Fehintola V Ajogbasile, Florette Treurnicht, Folarin Onikepe, Francine Ntoumi, Francisca M Muyembe, Frank E Z Ragomzingba, Fred A Dratibi, Fred-Akintunwa Iyanu, Gabriel K Mbunsu, Gaetan Thilliez, Gemma L Kay, George O Akpede, Gert U Zyl, Gordon A Awandare, Grace S Kpeli, Grit Schubert, Gugu P Maphalala, Hafaliana C Ranaivoson, Hannah E Omunakwe, Harris Onywera, Haruka Abe, Hela Karray, Hellen Nansumba, Henda Triki, AlbÃ©ric Adje Herve Kadjo, Hesham Elgahzaly, Hlanai Gumbo, Hota Mathieu, Hugo Kavunga-Membo, Ibtihel Smeti, Idowu B Olawoye, Ifedayo M O Adetifa, Ikponmwosa Odia, Ilhem Boutiba-Ben Boubaker, Iluoreh Ahmed Muhammad, Isaac Ssewanyana, Isatta Wurie, Iyaloo S Konstantinus, Jacqueline Wemboo Afiwa Halatoko, James Ayei, Janaki Sonoo, Jean-Claude C Makangara, Jean-Jacques M Tamfum, Jean-Michel Heraud, Jeffrey G Shaffer, Jennifer Giandhari, Jennifer Musyoki, Jerome Nkurunziza, Jessica N Uwanibe, Jinal N Bhiman, Jiro Yasuda, Joana Morais, Jocelyn Kiconco, John D Sandi, John Huddleston, John K Odoom, John M Morobe, John O Gyapong, John T Kayiwa, Johnson C Okolie, Joicymara S Xavier, Jones Gyamfi, Joseph F Wamala, Joseph H F Bonney, Joseph Nyandwi, Josie Everatt, Joweria Nakaseegu, Joyce M Ngoi, Joyce Namulondo, Judith U Oguzie, Julia C Andeko, Julius J Lutwama, Juma J H Mogga, Justin Oâ€™Grady, Katherine J Siddle, Kathleen Victoir, Kayode T Adeyemi, Kefentse A Tumedi, Kevin S Carvalho, Khadija Said Mohammed, Koussay Dellagi, Kunda G Musonda, Kwabena O Duedu, Lamia Fki-Berrajah, Lavanya Singh, Lenora M Kepler, Leon Biscornet, Leonardo de Oliveira Martins, Lucious Chabuka, Luicer Olubayo, Lul Deng Ojok, Lul Lojok Deng, Lynette I Ochola-Oyier, Lynn Tyers, Madisa Mine, Magalutcheemee Ramuth, Maha Mastouri, Mahmoud ElHefnawi, Maimouna Mbanne, Maitshwarelo I Matsheka, Malebogo Kebabonye, Mamadou Diop, Mambu Momoh, Lima Maria da Luz MendonÃ§a, Marietjie Venter, Marietou F Paye, Martin Faye, Martin M Nyaga, Mathabo Mareka, Matoke-Muhia Damaris, Maureen W Mburu, Maximillian G Mpina, Michael Owusu, Michael R Wiley, Mirabeau Y Tatfeng, Mitoha Ondoâ€™o Ayekaba, Mohamed Abouelhoda, Mohamed Amine Beloufa, Mohamed G Seadawy, Mohamed K Khalifa, Mooko Marethabile Matobo, Mouhamed Kane, Mounerou Salou, Mphaphi B Mbulawa, Mulenga Mwenda, Mushal Allam, My V T Phan, Nabil Abid, Nadine Rujeni, Nadir Abuzaid, Nalia Ismael, Nancy Elguindy, Ndeye Marieme Top, Ndongo Dia, NÃ©dio Mabunda, Nei-yuan Hsiao, Nelson BoricÃ³ Silochi, Ngiambudulu M Francisco, Ngonda Saasa, Nicholas Bbosa, Nickson Murunga, Nicksy Gumede, Nicole Wolter, Nikita Sitharam, Nnaemeka Ndodo, Nnennaya A Ajayi, NoÃ«l Tordo, Nokuzola Mbhele, Norosoa H Razanajatovo, Nosamiefan Iguosadolo, Nwando Mba, Ojide C Kingsley, Okogbenin Sylvanus, Oladiji Femi, Olubusuyi M Adewumi, Olumade Testimony, Olusola A Ogunsanya, Oluwatosin Fakayode, Onwe E Ogah, Ope-Ewe Oludayo, Ousmane Faye, Pamela Smith-Lawrence, Pascale Ondoa, Patrice Combe, Patricia Nabisubi, Patrick Semanda, Paul E Oluniyi, Paulo Arnaldo, Peter Kojo Quashie, Peter O Okokhere, Philip Bejon, Philippe Dussart, Phillip A Bester, Placide K Mbala, Pontiano Kaleebu, Priscilla Abechi, Rabeh El-Shesheny, Rageema Joseph, Ramy Karam Aziz, RenÃ© G Essomba, Reuben Ayivor-Djanie, Richard Njouom, Richard O Phillips, Richmond Gorman, Robert A Kingsley, Neto Rosa Maria D E S A Rodrigues, Rosemary A Audu, Rosina A A Carr, Saba Gargouri, Saber Masmoudi, Sacha Bootsma, Safietou Sankhe, Sahra Isse Mohamed, Saibu Femi, Salma Mhalla, Salome Hosch, Samar Kamal Kassim, Samar Metha, Sameh Trabelsi, Hassan Sara Agwa, Sarah Wambui Mwangi, Seydou Doumbia, Sheila Makiala-Mandanda, Sherihane Aryeetey, Shymaa S Ahmed, Side Mohamed Ahmed, Siham Elhamoumi, Sikhulile Moyo, Silvia Lutucuta, Simani Gaseitsiwe, Simbirie Jalloh, Soa Fy Andriamandimby, Sobajo Oguntope, SolÃ¨ne Grayo, Sonia Lekana-Douki, Sophie Prosolek, Soumeya Ouangraoua, Stephanie Wyk, Stephen F Schaffner, Stephen Kanyerezi, Steve Ahuka-Mundeke, Steven Rudder, Sureshnee Pillay, Susan Nabadda, Sylvie Behillil, Sylvie L Budiaki, Sylvie Werf, Tapfumanei Mashe, Thabo Mohale, Thanh Le-Viet, Thirumalaisamy P Velavan, Tobias Schindler, Tongai G Maponga, Trevor Bedford, Ugochukwu J Anyaneji, Ugwu Chinedu, Upasana Ramphal, Uwem E George, Vincent Enouf, Vishvanath Nene, Vivianne Gorova, Wael H Roshdy, Wasim Abdul Karim, William K Ampofo, Wolfgang Preiser, Wonderful T Choga, Yahaya Ali Ahmed, Yajna Ramphal, Yaw Bediako, Yeshnee Naidoo, Yvan Butera, Zaydah R Laurent, PGI Africa Africa Pathogen Genomics Initiative, Ahmed E O Ouma, Anne Gottberg, George Githinji, Matshidiso Moeti, Oyewale Tomori, Pardis C Sabeti, Amadou A Sall, Samuel O Oyola, Yenew T Kebede, Sofonias K Tessema, Tulio Oliveira, Christian Happi, Richard Lessells, John Nkengasong, Eduan Wilkinson' : '399 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">399 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Science</em>, Sep 2022
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.science.org/doi/10.1126/science.abq5358" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/The_evolving_SARS-CoV-2_epidemic_in_Africa-Insights_from_rapidly_expanding_genomic_surveillance.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1126/science.abq5358"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1126/science.abq5358" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The past 2 years, during which waves of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants swept the globe, have starkly highlighted health disparities across nations. Tegally et al. show how the coordinated efforts of talented African scientists have in a short time made great contributions to pandemic surveillance and data gathering. Their efforts and initiatives have provided early warning that has likely benefited wealthier countries more than their own. Genomic surveillance identified the emergence of the highly transmissible Beta and Omicron variants and now the appearance of Omicron sublineages in Africa. However, it is imperative that technology transfer for diagnostics and vaccines, as well the logistic wherewithal to produce and deploy them, match the data-gathering effort.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Nature</abbr></div>
<!-- Entry bib key -->
        <div id="pathogengenomics_2025" class="col-sm-8">
        <!-- Title -->
        <div class="title">Genomics reveals zoonotic and sustained human Mpox spread in West Africa</div>
        <!-- Author -->
        <div class="author">
        

        Edyth Parker,Â Ifeanyi F Omah,Â Doreen Delia Djuicy, and
          <span class="more-authors" title="click to view 59 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '59 more authors' ? 'Andrew Magee, Christopher H Tomkins-Tinch, James Richard Otieno, Patrick Varilly, Akeemat Opeyemi Ayinla, Ayotunde E Sijuwola, Muhammad I Ahmed, Oludayo O Ope-ewe, Olusola Akinola Ogunsanya, Alhaji Olono, Femi Mudasiru Saibu, Philomena Eromon, MoÃ¯se Henri Moumbeket Yifomnjou, Loique Landry Messanga Essengue, Martial Gides Wansi Yonga, Gael DieudonnÃ© Essima, Ibrahim Pascal Touoyem, Landry Jules Mouliem Mounchili, Sara Irene Eyangoh, Alain Georges Mballa Etoundi, Linda Esso, InÃ¨s Mandah Emah Nguidjol, Steve Franck Metomb, Cornelius Chebo, Samuel Mbah Agwe, Hans Makembe Mossi, Chanceline Ndongo Bilounga, Olusola Akanbi, Abiodun Egwuenu, Odianosen Ehiakhamen, Chimaobi Chukwu, Kabiru Suleiman, Afolabi Akinpelu, Adama Ahmad, Khadijah Isa Imam, Richard Ojedele, Victor Oripenaye, Kenneth Ikeata, Sophiyah Adelakun, Babatunde Olajumoke, Ãine Oâ€™Toole, Mark Zeller, Karthik Gangavarapu, Daniel J Park, Gerald Mboowa, Sofonias Kifle Tessema, Yenew Kebede Tebeje, Onikepe Folarin, Anise Happi, Philippe Lemey, Marc A Suchard, Kristian G Andersen, Pardis Sabeti, Andrew Rambaut, Chikwe Ihekweazu, Idris Jide, Ifedayo Adetifa, Richard Njouom, Christian T Happi' : '59 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">59 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nature</em>, May 2025
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-025-09128-2" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/Genomics_reveals_zoonotic_and_sustained_human_Mpox_spread_in_West_Africa.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-025-09128-2"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-025-09128-2" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Five years before the 2022 multi-country mpox outbreak, Nigeria and Cameroon reported their first cases in over three decades.1,2 While Nigeriaâ€™s outbreak is recognized as an ongoing human epidemic, the drivers of Cameroonâ€™s resurgence remain unclear.3,4 The rate of zoonoses remains uncertain in both countries, and gaps in genomic data obscure the timing, zoonotic and geographic origin of mpox virus (MPXV) emergence in humans. To address these uncertainties, we generated 118 MPXV genomes from Nigeria and Cameroon from 2018-2023. Our findings show that, in contrast to Nigeria, cases in Cameroon are the result of repeated zoonoses, with two distinct zoonotic lineages circulating across the Nigeria-Cameroon border. Our findings suggest that shared animal populations in the cross-border forest ecosystems drive virus emergence and spread. Accordingly, we identify the closest zoonotic outgroup to the Nigerian human epidemic lineage (hMPXV-1) in a southern Nigerian border state. We estimate that the shared ancestor of the zoonotic outgroup and hMPXV-1 circulated in animals in southern Nigeria in late 2013. We estimate that hMPXV-1 emerged in humans in August 2014 in the southern Rivers State and circulated undetected for three years. Rivers State acted as the main source of viral spread across the human epidemic. Our study sheds light on MPXVâ€™s recent establishment in the human population and highlights the risk of persistent zoonotic emergence of MPXV in the complex border regions of Cameroon and Nigeria.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Nature Medicine</abbr></div>
<!-- Entry bib key -->
        <div id="AfricaCDCdataplatform_2023" class="col-sm-8">
        <!-- Title -->
        <div class="title">A pan-African pathogen genomics data sharing platform to support disease outbreaks</div>
        <!-- Author -->
        <div class="author">
        

        Alan Christoffels,Â Gerald Mboowa,Â Peter Heusden, and
          <span class="more-authors" title="click to view 19 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '19 more authors' ? 'Sello Makhubela, George Githinji, Sarah Mwangi, Harris Onywera, Ndodo Nnaemeka, Gyamfi Daniel Amoako, Idowu Olawoye, Amadou Diallo, Placide Mbala-Kingebeni, Samuel O Oyola, Bright Adu, Christopher Mvelase, Pascale Ondoa, Athanasius Fred Dratibi, Abdourahmane Sow, Nicksy Gumede, Sofonias K Tessema, Ahmed Ogwell Ouma, Yenew Tebeje Kebede' : '19 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">19 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nature Medicine</em>, May 2023
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41591-023-02266-y" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/A_pan-African_pathogen_genomics_data_sharing_platform_to_support_disease_outbreaks.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41591-023-02266-y"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/s41591-023-02266-y" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>High-income countries have a wealth of genomics expertise that can be rapidly activated to deal with disease threats. African countries should invest in a federated data-management system for genomics epidemiology to deal with such threats better.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Cell</abbr></div>
<!-- Entry bib key -->
        <div id="mboowabarriersindatasharing_2024" class="col-sm-8">
        <!-- Title -->
        <div class="title">Africa in the era of pathogen genomics: Unlocking data barriers</div>
        <!-- Author -->
        <div class="author">
        

        Gerald Mboowa,Â Sofonias K Tessema,Â Alan Christoffels, and
          <span class="more-authors" title="click to view 3 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '3 more authors' ? 'Nicaise Ndembi, Yenew Tebeje Kebede, Jean Kaseya' : '3 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">3 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Cell</em>, Sep 2024
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.cell.com/cell/abstract/S0092-8674(24)00960-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/Africa_in_the_era_of_pathogen_genomics_Unlocking_data_barriers.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.cell.2024.08.032"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2024.08.032" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Rapid expansion of pathogen sequencing capacity in Africa has led to a paradigm shift from relying on others to locally generating genomic data and sharing it with the global community. However, several barriers remain to be unlocked for timely processing, analysis, dissemination, and effective use of pathogen sequence data for pandemic prevention, preparedness, and response.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Nat Rev Gen</abbr></div>
<!-- Entry bib key -->
        <div id="CRISPR_2025" class="col-sm-8">
        <!-- Title -->
        <div class="title">A clinical milestone for CRISPR in sickle-cell disease : Genome editing</div>
        <!-- Author -->
        <div class="author">
        

        Gerald Mboowa</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nature Reviews Genetics</em>, Sep 2025
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41576-025-00898-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/A_clinical_milestone_for_CRISPR_in_sickle-cell_disease.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41576-025-00898-z"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/s41576-025-00898-z" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The haemoglobinopathies transfusion-dependent Î²-thalassemia (TDT) and sickle-cell disease (SCD) are two globally prevalent and severe monogenic blood disorders. These conditions impose substantial morbidity and mortality, and their burden falls disproportionately on populations in sub-Saharan Africa, the Middle East and parts of Asia, underscoring the global health significance of any curative approach. In 2021, a study by Frangoul et al. marked a defining moment for genomic medicine and global health. Published in The New England Journal of Medicine, the authors reported the clinical application of CRISPRâ€“Cas9 genome editing to treat two patients with TDT or SCD.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">BMC Gen</abbr></div>
<!-- Entry bib key -->
        <div id="nsubugageneralizability_2024" class="col-sm-8">
        <!-- Title -->
        <div class="title">Generalizability of machine learning in predicting antimicrobial resistance in <i>E. coli</i>: a multi-country case study in Africa</div>
        <!-- Author -->
        <div class="author">
        

        Mike Nsubuga,Â Ronald Galiwango,Â Daudi Jjingo, and
          <span class="more-authors" title="click to view 1 more author" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '1 more author' ? 'Gerald Mboowa' : '1 more author';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">1 more author</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>BMC Genomics</em>, Mar 2024
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://link.springer.com/article/10.1186/s12864-024-10214-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/Generalizability_of_machine_learning_in_predicting_antimicrobial_resistance_in_E_coli-a_multi-country_case_study_in_Africa.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1186/s12864-024-10214-4"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-024-10214-4" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Antimicrobial resistance (AMR) remains a significant global health threat particularly impacting low- and middle-income countries (LMICs). These regions often grapple with limited healthcare resources and access to advanced diagnostic tools. Consequently, there is a pressing need for innovative approaches that can enhance AMR surveillance and management. Machine learning (ML) though underutilized in these settings, presents a promising avenue. This study leverages ML models trained on whole-genome sequencing data from England, where such data is more readily available, to predict AMR in E. coli, targeting key antibiotics such as ciprofloxacin, ampicillin, and cefotaxime. A crucial part of our work involved the validation of these models using an independent dataset from Africa, specifically from Uganda, Nigeria, and Tanzania, to ascertain their applicability and effectiveness in LMICs.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Nat Comm Med</abbr></div>
<!-- Entry bib key -->
        <div id="kidenyaprs_2024" class="col-sm-8">
        <!-- Title -->
        <div class="title">Inclusiveness of the All of Us Research Program improves polygenic risk scores and fosters genomic medicine for all</div>
        <!-- Author -->
        <div class="author">
        

        Benson R Kidenya,Â andÂ Gerald Mboowa</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Communications Medicine</em>, Nov 2024
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s43856-024-00647-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/Inclusiveness_of_the_All_of_Us_Research_Program_improves_polygenic_risk_scores_and_fosters_genomic_medicine_for_all.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s43856-024-00647-z"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/s43856-024-00647-z" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Polygenic risk scores (PRS) show promise but have accuracy disparities across ancestries due to underrepresentation in the existing genomic databases. Here, we outline the initiative of All of Us Research Program in refining PRS and advancing genomic medicine for all.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">PLOS Biology</abbr></div>
<!-- Entry bib key -->
        <div id="mpox_2022" class="col-sm-8">
        <!-- Title -->
        <div class="title">Urgent need for a non-discriminatory and non-stigmatizing nomenclature for monkeypox virus</div>
        <!-- Author -->
        <div class="author">
        

        Christian Happi,Â Ifedayo Adetifa,Â Placide Mbala, and
          <span class="more-authors" title="click to view 30 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '30 more authors' ? 'Richard Njouom, Emmanuel Nakoune, Anise Happi, Nnaemeka Ndodo, Oyeronke Ayansola, Gerald Mboowa, Trevor Bedford, Richard A Neher, Cornelius Roemer, Emma Hodcroft, Houriiyah Tegally, AÂ´ine Oâ€™Toole, Andrew Rambaut, Oliver Pybus, Moritz U G Kraemer, Eduan Wilkinson, Joana Isidro, VÄ±Â´tor Borges, Miguel Pinto, Paulo JoÃ£o Gomes, Lucas Freitas, Paola C Resende, Raphael T C Lee, Sebastian Maurer-Stroh, Cheryl Baxter, Richard Lessells, Ahmed E Ogwell, Yenew T Kebede, Sofonias K Tessema, Tulio Oliveira' : '30 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">30 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>PLOS Biology</em>, Nov 2022
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001769" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/Urgent_need_for_a_non-discriminatory_and_non-stigmatizing_nomenclature_for_monkeypox_virus.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1371/journal.pbio.3001769"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001769" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>We propose a novel, non-discriminatory classification of monkeypox virus diversity. Together with the World Health Organization, we named three clades (I, IIa and IIb) in order of detection. Within IIb, the cause of the current global outbreak, we identified multiple lineages (A.1, A.2, A.1.1 and B.1) to support real-time genomic surveillance.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Crit Care Med</abbr></div>
<!-- Entry bib key -->
        <div id="Tuberculosisprediction_2012" class="col-sm-8">
        <!-- Title -->
        <div class="title">Cough aerosols of <i>Mycobacterium tuberculosis</i> predict new infection: a household contact study</div>
        <!-- Author -->
        <div class="author">
        

        Edward C Jones-LÃ³pez,Â Olive Namugga,Â Francis Mumbowa, and
          <span class="more-authors" title="click to view 11 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '11 more authors' ? 'Martin Ssebidandi, Olive Mbabazi, Stephanie Moine, Gerald Mboowa, Matthew P Fox, Nancy Reilly, Irene Ayakaka, Soyeon Kim, Alphonse Okwera, Moses Joloba, Kevin P Fennelly' : '11 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">11 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>American Journal of Respiratory and Critical Care Medicine</em>, Dec 2012
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.atsjournals.org/doi/10.1164/rccm.201208-1422OC" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/moine-et-al-2013-cough-aerosols-of-mycobacterium-tuberculosis-predict-new-infection-a-household-contact-study.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1164/rccm.201208-1422OC"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1164/rccm.201208-1422OC" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Rationale: Airborne transmission of Mycobacterium tuberculosis results from incompletely characterized host, bacterial, and environmental factors. Sputum smear microscopy is associated with considerable variability in transmission. Objectives: To evaluate the use of cough-generated aerosols of M. tuberculosis to predict recent transmission. Methods: Patients with pulmonary tuberculosis (TB) underwent a standard evaluation and collection of cough aerosol cultures of M. tuberculosis. We assessed household contacts for new M. tuberculosis infection. We used multivariable logistic regression analysis with cluster adjustment to analyze predictors of new infection. Measurements and main results: From May 2009 to January 2011, we enrolled 96 sputum culture-positive index TB cases and their 442 contacts. Only 43 (45%) patients with TB yielded M. tuberculosis in aerosols. Contacts of patients with TB who produced high aerosols (â‰¥10 CFU) were more likely to have a new infection compared with contacts from low-aerosol (1-9 CFU) and aerosol-negative cases (69%, 25%, and 30%, respectively; P = 0.009). A high-aerosol patient with TB was the only predictor of new M. tuberculosis infection in unadjusted (odds ratio, 5.18; 95% confidence interval, 1.52-17.61) and adjusted analyses (odds ratio, 4.81; 95% confidence interval, 1.20-19.23). Contacts of patients with TB with no aerosols versus low and high aerosols had differential tuberculin skin test and interferon-Î³ release assay responses. Conclusions: Cough aerosols of M. tuberculosis are produced by a minority of patients with TB but predict transmission better than sputum smear microscopy or culture. Cough aerosols may help identify the most infectious patients with TB and thus improve the cost-effectiveness of TB control programs.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Microb Genomics</abbr></div>
<!-- Entry bib key -->
        <div id="microbialgenomics_2021" class="col-sm-8">
        <!-- Title -->
        <div class="title">
<b><i>rMAP</i></b>Â : the Rapid Microbial Analysis Pipeline for <i>ESKAPE</i>Â   bacterial group whole-genome sequence data</div>
        <!-- Author -->
        <div class="author">
        

        Ivan Sserwadda,Â andÂ Gerald Mboowa</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Microbial Genomics</em>, Jun 2021
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000583" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/rMAP-the-Rapid_Microbial_Analysis_Pipeline_for_ESKAPE_bacterial_group_whole-genome_sequence_data.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1099/mgen.0.000583"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1099/mgen.0.000583" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>The recent re-emergence of multidrug-resistant pathogens has exacerbated their threat to worldwide public health. The evolution of the genomics era has led to the generation of huge volumes of sequencing data at an unprecedented rate due to the ever-reducing costs of whole-genome sequencing (WGS). We have developed the Rapid Microbial Analysis Pipeline (rMAP), a user-friendly pipeline capable of profiling the resistomes of ESKAPE pathogens ( Enterococcus faecium , Staphylococcus aureus , Klebsiella pneumoniae , Acinetobacter baumannii , Pseudomonas aeruginosa and Enterobacter species) using WGS data generated from Illuminaâ€™s sequencing platforms. rMAP is designed for individuals with little bioinformatics expertise, and automates the steps required for WGS analysis directly from the raw genomic sequence data, including adapter and low-quality sequence read trimming, de novo genome assembly, genome annotation, single-nucleotide polymorphism (SNP) variant calling, phylogenetic inference by maximum likelihood, antimicrobial resistance (AMR) profiling, plasmid profiling, virulence factor determination, multi-locus sequence typing (MLST), pangenome analysis and insertion sequence characterization (IS). Once the analysis is finished, rMAP generates an interactive web-like html report. rMAP installation is very simple, it can be run using very simple commands. It represents a rapid and easy way to perform comprehensive bacterial WGS analysis using a personal laptop in low-income settings where high-performance computing infrastructure is limited.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Nature</abbr></div>
<!-- Entry bib key -->
        <div id="africacdc_2024" class="col-sm-8">
        <!-- Title -->
        <div class="title">The pandemic agreement: an African perspective</div>
        <!-- Author -->
        <div class="author">
        

        Nicaise Ndembi,Â Gerald Mboowa,Â Sofonias K Tessema, and
          <span class="more-authors" title="click to view 2 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '2 more authors' ? 'Yenew Kebede Tebeje, Jean Kaseya' : '2 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">2 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nature</em>, Jul 2024
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12188460/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/The_pandemic_agreement_an_African_perspective.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/d41586-024-02317-5"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/d41586-024-02317-5" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Your Editorial on efforts to agree a global pandemic treaty (Nature 629, 727; 2024) emphasizes that genomic surveillance and pathogen and data sharing are key to effective infection control and development of therapies. With investment spurred by COVID-19, more than 70% of African Union (AU) member states have developed local capacity to sequence pathogen genomes (M. Makoni Lancet Microbe 1, e318; 2020), and more than 170,000 genomes of the virus SARS-CoV-2 have been sequenced and shared. In collaboration with AU member states, the Africa Centres for Disease Control and Prevention (CDC) is creating a biobanking network across the continent to address pathogen sharing. Led by the AU and the Africa CDC, a Common African Position on Pandemic Prevention, Preparedness, and Response was adopted in February 2024. Agreement on a pathogen-access and benefit-sharing system â€” which will allow rapid exchange of sequence data and equitable and timely access to countermeasures for pandemic preparedness and response â€” will be the most crucial outcome of the pandemic treaty. Sharing of pathogen data, reciprocated with multilateral benefits, can provide a guarantee of safety and equity for people all over the world, whether in higher- or lower-income countries.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Nature Comm</abbr></div>
<!-- Entry bib key -->
        <div id="africacdc_2026" class="col-sm-8">
        <!-- Title -->
        <div class="title">Multicountry genomic analysis underscores regional cholera spread in Africa</div>
        <!-- Author -->
        <div class="author">
        

        Gerald Mboowa,Â Nathaniel Lucero Matteson,Â Collins Kipngetich Tanui, and
          <span class="more-authors" title="click to view 51 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '51 more authors' ? 'Mpanga Kasonde, Guyguy Kusanzangana Kamwiziku, Olusola Anuoluwapo Akanbi, JucunÃº Johane Elias Chitio, Mathews Kagoli, RenÃ© Ghislain Essomba, Alisen Ayitewala, Isaac Ssewanyana, Valentina Josiane Ngo Bitoungui, Adrienne Aziza Amuri, Andrew S Azman, Olajumoke Atinuke Babatunde, Blaise Mboringong Akenji, AnaÃ¯s Broban, Espoir Bwenge Malembaka, Francis Ongole, Chimaobi Emmanuel Chukwu, NÃ¡lia Ismael, Otridah Kapona, Osvaldo Laurindo, Placide Kingebeni Mbala, Georges Alain Etoundi Mballa, Imelda Carlos Zulfa Miambo, Alex Ansaye Mwanyongo, Grace Najjuka, Joseph Mutale, Kunda Musonda, Allan Muruta Niyonzima, Mirriam Ethel Nyenje, Michael Popoola, Doreen Mainza Shempela, Christiane Medi Sike, SofiÃ£o Manjor Sitoe, Dorcas Waruguru Wanjohi, Placide Okitayemba Welo, Mtisunge Yelewa, Sebastian Yennan, Lucius Ziba, CholGEN Consortium, Joseph Ephram Bitilinyu-Bangoh, Roma Chilengi, Hamsatou Hadja, Jide Idris, JosÃ© Paulo MaurÃ­cio Langa, Daniel Mukadi-Bamuleka, Susan Nabadda, Amanda K Debes, David A Sack, Yenew Kebede Tebeje, Shirlee Wohl, Sofonias Kifle Tessema' : '51 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">51 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Nature Communications</em>, Feb 2026
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41467-026-68642-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/Multicountry_genomic_analysis_underscores_regional_cholera_spread_in_Africa.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41467-026-68642-7"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-026-68642-7" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Cholera remains a significant public health burden in many countries in sub-Saharan Africa, though the exact mechanisms of bacterial emergence and spread remain largely undefined. We generated genomic data from 763 Vibrio cholerae O1 isolates predominantly collected between 2019-2024 to create the largest dataset of V. cholerae genomes sequenced locally in Africa. This dataset enabled us to interrogate recent patterns of spread, including the rapid circulation of the AFR15 lineage associated with unusually large outbreaks in Southern Africa. We provide evidence for the movement of the AFR15 lineage into new African Union Member States and confirm previously observed differences in V. cholerae transmission dynamics in West versus East Africa, though cross-border transmission is prevalent on both sides of the continent. Despite observed differences, evolutionary processes are similar across lineages and we find no evidence for significant changes in antimicrobial resistance genotypes. Overall, our findings emphasize the importance of regionally coordinated cross-border surveillance and interventions, while also demonstrating the critical role of locally generated genomic data in understanding the spread of cholera in Africa.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Bioinform Adv</abbr></div>
<!-- Entry bib key -->
        <div id="amr_2028" class="col-sm-8">
        <!-- Title -->
        <div class="title">
<b><i>rMAP-2.0</i></b>Â : A modular, reproducible and scalable WDL-Cromwell-Docker workflow for genomic analysis of <i>ESKAPEE</i>Â  pathogens</div>
        <!-- Author -->
        <div class="author">
        

        Gerald Mboowa,Â Ivan Sserwadda,Â andÂ Stephen Kanyerezi</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Bioinformatics Advances</em>, Feb 2026
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://academic.oup.com/bioinformaticsadvances/advance-article/doi/10.1093/bioadv/vbag046/8483022" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/rMAP_2.0_Amodular_reproducible_and_scalable_WDL_Cromwell_Docker_workflow_for_genomic_analysis_of_ESKAPEE_pathogens.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1093/bioadv/vbag046"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1093/bioadv/vbag046" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Motivation: Antimicrobial resistance surveillance in ESKAPEE pathogens requires reproducible, portable whole-genome analysis that public health laboratories including those operating under data-sovereignty constraints can run on laptops, institutional servers, or cloud backends without local dependency conflicts. rMAP 2.0 addresses these needs using a containerized Workflow Description Language pipeline executed with Cromwell. Results:
rMAP 2.0 standardizes end-to-end bacterial whole-genome analysisâ€”read quality control, trimming, assembly and annotation, resistance/virulence/mobile-element profiling, sequence typing, pangenome inference, and phylogenetic reconstruction using containerized execution, and generates a single interactive HTML report that collates outputs for rapid review. The workflow supports fully offline execution (including BLAST searches) for data-sovereign deployments and can run on local workstations, institutional servers, and cloud backends where Docker is supported, providing a consistent execution environment without local tool installation. In a representative benchmark of 20 Enterobacterales isolates, rMAP 2.0 completed a cohort run in âˆ¼4.5â€‰hours on an 8-core/16-GB laptop and flagged a public record misannotated in public repository metadata (SRR9703249, reclassified from Klebsiella pneumoniae to Enterobacter cloacae sequence type 182), while confirming lineage assignments such as Escherichia coli sequence type 131. Availability &amp; implementation: https://github.com/gmboowa/rMAP-2.0 and example workflow reports are available at: https://gmboowa.github.io/rMAP-2.0/.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Sci Reports</abbr></div>
<!-- Entry bib key -->
        <div id="Stephen_PhD_2026" class="col-sm-8">
        <!-- Title -->
        <div class="title">Targeted metagenomics reveals hidden chickenpox epidemic amid Mpox surveillance in Uganda</div>
        <!-- Author -->
        <div class="author">
        

        Stephen Kanyerezi,Â Alisen Ayitewala,Â Jupiter Marina Kabahita, and
          <span class="more-authors" title="click to view 26 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '26 more authors' ? 'Hellen Rosette Oundo, Julius Sseruyange, Tenywa Wilson, Godwin Tusabe, Stacy Were, Moses Murungi, Martha Nabukyu, Valeria Nakintu, Caroline Makoha, Ivan Sserwadda, Harris Onywera, Collins Tanui, Ibrahim Mugerwa, Atek Kagirita, Benard Lubwama, Eilu Roggers Michael, David Patrick Kateete, Morgan Otita, Samuel Giduddu, Daudi Jjingo, Andrew Nsawotebba, Gerald Mboowa, Aloysious Ssemaganda, Susan Nabadda, Sofonias K Tessema, Isaac Ssewanyana' : '26 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">26 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Scientific Reports</em>, Feb 2026
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41598-026-38778-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/Targeted_metagenomics_reveals_hidden_chickenpox_epidemic_amid_Mpox_surveillance_in_Uganda.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41598-026-38778-z"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-026-38778-z" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>In regions where both monkeypox virus (MPXV) and varicella zoster virus (VZV) are co-circulating, overlapping clinical manifestations can complicate clinical diagnosis. During the MPXV outbreak declared in Uganda in July, 2024, symptomatic suspected cases tested PCR negative for Mpox. To determine the cause of symptoms, we employed metagenomic sequencing with a targeted Viral Surveillance panel in 284 MPXV negative samples. VZV was identified as the predominant pathogen in 86% of MPXV-negative cases, suggesting a concurrent chickenpox surge. Using the VaricellaGen pipeline for variant calling, clade typing, and phylogeny, 118 (49%) samples that achievedâ€‰â‰¥â€‰70% genome coverage were of clade 5 based on the single-nucleotide polymorphism (SNP) dataset. This data confirms co-circulation of VZV during the MPXV outbreak in Uganda. Our results underscore the need for laboratory confirmation of MPXV and the inclusion of VZV in the testing algorithm during the Mpox outbreak.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
<div class="col-sm-2 abbr"><abbr class="badge">Brief Bioinform</abbr></div>
<!-- Entry bib key -->
        <div id="Capacity_building_2026" class="col-sm-8">
        <!-- Title -->
        <div class="title">Pathways, outputs and impact of NIH-supported bioinformatics and genomics graduate trainees in Africa</div>
        <!-- Author -->
        <div class="author">
        

        Daudi Jjingo,Â Andrew Walakira,Â Suhaila Hashim, and
          <span class="more-authors" title="click to view 64 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '64 more authors' ? 'Cisse Cheickna, Ronald Galiwango, Caleb Kibet, Florence N Kivunike, Gerald Mboowa, Fredrick Elishama Kakembo, Babajide Ayodele, Jean-Baka Domelevo Entfellner, Santie Villiers, Karen Wambui, Segun Fatumo, Tinashe Chikowore, John Mukisa, Alfred Ssekagiri, Nicholas Bbosa, Julius Mulindwa, Samuel Kyobe, Mike Nsubuga, Grace Kebirungi, Eric Katagirya, Savannah Mwesigwa, Ibra Lujumba, Rogers Kamulegeya, Samuel Kirimunda, Stephen Kanyerezi, Shahiid Kiyaga, Ivan Sserwadda, Davis Kiberu, Bernard S Bagaya, Julius Okwir, Patricia Nabisubi, Grace Nabakooza, Mugume Twinamatsiko Atwine, Ricard Sserunjogi, Rolanda Julius, Mariam QuiÃ±ones, Meghan McCarthy, Phillip Cruz, Karlynn Noble, Christopher J Whalen, Darrell Hurt, Maria Y Giovanni, Michael Tartakovsky, Deogratius Ssemwanga, John M Kitayimbwa, Steven J Reynolds, Christopher C Whalen, Andrew Kambugu, Neil A Hanchard, Li Jian, Peter Amoako-Yirenkyi, Graeme Mardon, I King Jordan, Samson Pandam Salifu, Mamadou Wele, Ezekiel Adebiyi, Jeffrey G Shaffer, Seydou Doumbia, David Patrick Kateete, Michelle Skelton, Nicola Mulder, Jonathan K Kayondo, Daniel Masiga, H3Africa Consortium' : '64 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '10');
              ">64 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Briefings in Bioinformatics</em>, Feb 2026
        </div>
        <div class="periodical">
          
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://academic.oup.com/bib/article/27/1/bbag046/8475545" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener noopener noreferrer" target="_blank">HTML</a>
            <a href="/assets/pdf/Pathways_outputs_and_impact_of_NIH_supported_bioinformatics_and_genomics_graduate_trainees_in_Africa.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a>
          </div>
          
          
          
          
          <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1093/bib/bbag046"></span>
              <span class="__dimensions_badge_embed__" data-doi="10.1093/bib/bbag046" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Global biomedical and health research is increasingly relying on genomic and computational approaches, largely driven by the increasing volumes of nucleic acid sequencing. Concurrently, epidemiological studies and clinical records are generating enormous amounts of data amenable to disease modeling, machine learning, and artificial intelligence techniques. Bioinformatics and data science expertise is therefore essential for improved population health. Accordingly, in 2012, the US National Institutes of Health (NIH) in partnership with the Wellcome Trust, and with support from the African Society for Human Genetics, initiated the H3Africa (Human Heredity and Health in Africa) consortium. One of its key goals was to build capacity among African scientists to lead research on genetic and environmental contributors to health and disease across the continent. In 2017, the NIH provided funding to support the establishment of four graduate bioinformatics training programs across five African universities. Over seven years, these programs enrolled multiple trainees (nâ€‰&gt;â€‰270), with &gt;110 earning Masterâ€™s degrees and &gt;20 completing PhDs in Bioinformatics. It is thus timely to evaluate the outcomes and impact of these programs, particularly regarding graduation rates, career trajectories, and the institutions and research domains their alumni are serving. We also assess employment outcomes and the nature of the research they are enabling (nâ€‰&gt;â€‰110 peer-reviewed articles). We additionally include the progress and outputs of the programsâ€™ instructors, which were partially enabled by program resources, networks, and trainees. Overall, this review paints valuable insights into the pioneering role of NIH extramural support in shaping Africaâ€™s biomedical research landscape.</p>
          </div>
        </div>
      </div>
</li>
</ol>
          </div>


          <!-- Social -->
            <div class="social">
              <div class="contact-icons">
                <a href="mailto:%67%6D%62%6F%6F%77%61@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fas fa-envelope"></i></a>
            <a href="https://orcid.org/0000-0001-8445-9414" title="ORCID" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="ai ai-orcid"></i></a>
            <a href="https://scholar.google.com/citations?user=P9dSDhoAAAAJ" title="Google Scholar" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="ai ai-google-scholar"></i></a>
            <a href="https://www.researchgate.net/profile/Gerald-Mboowa/" title="ResearchGate" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="ai ai-researchgate"></i></a>
            <a href="https://github.com/gmboowa" title="GitHub" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="fab fa-github"></i></a>
            <a href="https://www.linkedin.com/in/gerald-mboowa-99020537" title="LinkedIn" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="fab fa-linkedin"></i></a>
            <a href="https://twitter.com/GeraldGmboowa" title="Twitter" rel="external nofollow noopener noopener noreferrer" target="_blank"><i class="fab fa-twitter"></i></a>
            

              </div>

              <div class="contact-note">
                The best way to reach me is through my personal email gmboowa@gmail.com

              </div>

            </div>
        </article>

</div>

      
    </div>

    <!-- Footer -->    <footer class="sticky-bottom mt-5">
      <div class="container">
        Â© Copyright 2026 Gerald  Mboowa. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener noopener noreferrer" target="_blank">al-folio</a> theme.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="/assets/js/bootstrap.bundle.min.js"></script>
  <!-- <script src="/assets/js/mdb.min.js"></script> -->
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script>

  <!-- Bootstrap Table -->
  <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script>

  <!-- Load Common JS -->
  <script src="/assets/js/no_defer.js"></script>
  <script defer src="/assets/js/common.js"></script>
  <script defer src="/assets/js/copy_code.js" type="text/javascript"></script>

    
  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
  <script async src="https://badge.dimensions.ai/badge.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id="></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){ window.dataLayer.push(arguments); }
    gtag('js', new Date());
    gtag('config', '');
  </script>
    

<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  /*
   * This JavaScript code has been adapted from the article 
   * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar, 
   * published on the website https://css-tricks.com on the 7th of May, 2014.
   * Couple of changes were made to the original code to make it compatible 
   * with the `al-foio` theme.
   */
  const progressBar = $("#progress");
  /*
   * We set up the bar after all elements are done loading.
   * In some cases, if the images in the page are larger than the intended
   * size they'll have on the page, they'll be resized via CSS to accomodate
   * the desired size. This mistake, however, breaks the computations as the
   * scroll size is computed as soon as the elements finish loading.
   * To account for this, a minimal delay was introduced before computing the
   * values.
   */
  window.onload = function () {
    setTimeout(progressBarSetup, 50);
  };
  /*
   * We set up the bar according to the browser.
   * If the browser supports the progress element we use that.
   * Otherwise, we resize the bar thru CSS styling
   */
  function progressBarSetup() {
    if ("max" in document.createElement("progress")) {
      initializeProgressElement();
      $(document).on("scroll", function() {
        progressBar.attr({ value: getCurrentScrollPosition() });
      });
      $(window).on("resize", initializeProgressElement);
    } else {
      resizeProgressBar();
      $(document).on("scroll", resizeProgressBar);
      $(window).on("resize", resizeProgressBar);
    }
  }
  /*
   * The vertical scroll position is the same as the number of pixels that
   * are hidden from view above the scrollable area. Thus, a value > 0 is
   * how much the user has scrolled from the top
   */
  function getCurrentScrollPosition() {
    return $(window).scrollTop();
  }

  function initializeProgressElement() {
    let navbarHeight = $("#navbar").outerHeight(true);
    $("body").css({ "padding-top": navbarHeight });
    $("progress-container").css({ "padding-top": navbarHeight });
    progressBar.css({ top: navbarHeight });
    progressBar.attr({
      max: getDistanceToScroll(),
      value: getCurrentScrollPosition(),
    });
  }
  /*
   * The offset between the html document height and the browser viewport
   * height will be greater than zero if vertical scroll is possible.
   * This is the distance the user can scroll
   */
  function getDistanceToScroll() {
    return $(document).height() - $(window).height();
  }

  function resizeProgressBar() {
    progressBar.css({ width: getWidthPercentage() + "%" });
  }
  // The scroll ratio equals the percentage to resize the bar
  function getWidthPercentage() {
    return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
  }
</script>

    

    <!-- Theme Toggle -->
    <script>
      document.addEventListener("DOMContentLoaded", function () {
          const toggleSwitch = document.getElementById("theme-toggle");
          const currentTheme = localStorage.getItem("theme") || "light";
          document.documentElement.setAttribute("data-theme", currentTheme);

          toggleSwitch.addEventListener("click", function () {
              const theme = document.documentElement.getAttribute("data-theme");
              const newTheme = theme === "dark" ? "light" : "dark";
              document.documentElement.setAttribute("data-theme", newTheme);
              localStorage.setItem("theme", newTheme);
          });
      });
    </script>

  </body>
</html>
